example/lib/edge.dart

```diff
@@ -0,0 +1,85 @@
+import 'dart:developer';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_aepedge/flutter_aepedge_data.dart';
+import 'package:flutter_aepedge/flutter_aepedge.dart';
+import 'util.dart';
+
+class EdgePage extends StatefulWidget {
+  @override
+  _MyAppState createState() => _MyAppState();
+}
+
+class _MyAppState extends State<EdgePage> {
+  String _edgeVersion = 'Unknown';
+
+  List<EventHandle> _edgeEventHandleResponse = List.empty();
+  @override
+  void initState() {
+    super.initState();
+    initPlatformState();
+  }
+
+  // Platform messages are asynchronous, so we initialize in an async method.
+  Future<void> initPlatformState() async {
+    late String edgeVersion;
+    // Platform messages may fail, so we use a try/catch PlatformException.
+    try {
+      edgeVersion = await Edge.extensionVersion;
+    } on PlatformException {
+      log("Failed to get extension versions");
+    }
```

Comment:
I don't see an error being thrown for the extensionVersion API, can we remove the try/catch here too?

Reply:
removed